.github/workflows/cs.yml

```
@@ -68,6 +73,11 @@ jobs:
       - name: Check XML code style
         run: diff -B ./PHPCSDebug/ruleset.xml <(xmllint --format "./PHPCSDebug/ruleset.xml")
 
+      # Validate the Docs XML file(s).
+      # @link http://xmlsoft.org/xmllint.html
+      - name: Validate docs against schema
+        run: xmllint --noout --schema DocsXsd/phpcsdocs.xsd ./PHPCSDebug/Docs/**/*.xml
```

```suggestion
        run: xmllint --noout --schema DocsXsd/phpcsdocs.xsd ./PHPCSDebug/Docs/*/*Standard.xml
```

While the current command works on Linux, it does not seem work on Windows. The above does work on Windows too.

While this isn't really an issue for the CI, I know I often look at the CI for packages to check what checks to run and how to run those locally. I imagine other people may too, so having a cross-OS compatible command here helps to document that (for lack of a CONTRIBUTING file for now).